src/pages/get-started/installation.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Install Adobe I/O Events for Adobe Commerce
+description: Learn how to install the Commerce modules needed for Adobe I/O Events for Adobe Commerce.
+---
+
+# Install and configure Adobe I/O Events for Adobe Commerce
+
+Adobe I/O Events for Adobe Commerce requires Adobe Commerce 2.4.5 or later. Magento Open Source is not supported.
+
+After you have created an [App Builder project](./project-setup.md), you must install the Commerce modules that enable integrations with Adobe I/O Events.
+
+## Install Adobe I/O modules on Commerce
+
+The following steps apply to both Adobe Commerce on cloud infrastructure and on-premises installations. Cloud customers must perform additional steps to configure the `ece-tools` package.
+
+1. If you are running Commerce 2.4.5, use the following command to load the eventing modules:
+
+   ```bash
+   composer require magento/commerce-eventing=^1.0
+   ```
+
+   Commerce 2.4.6 beta and later loads these modules automatically.
+
+1. Update the project dependencies.
+
+  ```bash
+  composer update
+  ```
+
+1. Run the following command to generate the `AdobeCommerceEvents` module. This module helps register events. Initially, the module is empty.
```

Comment:
It's not necessary at this point to generate a module as there are no registered events.
```
This module helps register events.
```
This module consists of generated plugins based on a list of subscribed events, maybe better to say it helps to publish or process events.